doc/source/whatsnew/v1.5.0.rst

```diff
@@ -874,6 +874,7 @@ I/O
 - Bug in :class:`StataWriter` where value labels were always written with default encoding (:issue:`46750`)
 - Bug in :class:`StataWriterUTF8` where some valid characters were removed from variable names (:issue:`47276`)
 - Bug in :meth:`DataFrame.to_excel` when writing an empty dataframe with :class:`MultiIndex` (:issue:`19543`)
+- Bug in :func:`read_xml` when reading XML with duplicate element and attribute names (:issue:`47343`)
```

Comment:
Just to confirm, this PR fixes a bug that is separate from https://github.com/pandas-dev/pandas/pull/45724 which will come out in 1.5?

If this fixes something related to https://github.com/pandas-dev/pandas/pull/45724, I don't think we need a whatsnew note here